### PR TITLE
cai nowicki pull request

### PR DESCRIPTION
**Pull request by CaiNowicki**

### CODE REVIEW

**Comment by llpk79** [comment]
## Applied Modeling

 - [x] Choose your target. Which column in your tabular dataset will you predict?
 - [x] Is your problem regression or classification?
 - [x] How is your target distributed?
 - [x] Choose which observations you will use to train, validate, and test your model.
 - [ ] Choose your evaluation metric(s).
 - [ ] Begin to clean and explore your data.
 - [x] Begin to choose which features, if any, to exclude. Would some features "leak" future information?

**Comment by llpk79** [comment]
## Permutation & Boosting

 - [ ] If you haven't completed assignment #1, please do so first.
 - [x] Continue to clean and explore your data. Make exploratory visualizations.
    - I think we can explain things a little more clearly than this.
 - [x] Fit a model. Does it beat your baseline?
    - Having trouble following the notebook. Functions are used before they're defined, hyperparameters are searched for, but I'm not sure they're used. Clean it up!
 - [x] Try xgboost.
 - [x] Get your model's permutation importances.

**Review by llpk79 — APPROVED** [review]
Much better. Still need to work on making sure the notebook runs before committing. It's difficult to know what needs to be changed when you haven't created the thing. 😄

**Comment by llpk79** [comment]
And make sure you're committing to the proper repo and branch.
